docs/reference/service-discovery.md

```diff
@@ -0,0 +1,139 @@
+# Service Discovery Service
```

Comment:
We agreed on naming it Discovery Service. As Service Discovery Service sounds funny